packages/core/src/common/_mixins.scss

```diff
@@ -103,6 +103,39 @@ $pt-dark-intent-text-colors: (
   @include force-hardware-acceleration();
 }
 
+// A collection of mixins that utilize the CSS Flexbox model to standardize
+// the layout and composition of Blueprint components.
+
+// pt-flex-parent styles
+@mixin pt-flex-parent() {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+}
+
+@mixin pt-flex-parent-fill() {
+  width: 100%;
+}
+
+@mixin pt-flex-parent-direction-horizontal {
+  flex-direction: row;
+}
+
+@mixin pt-flex-parent-direction-vertical {
+  flex-direction: column;
+}
+
+// pt-flex-child styles
```

Comment:
remove unnecessary comment (and line 109): not helpful as divider, and immediately obvious from actual code.

Reply:
yep